logos/src/logos/classification/classification_manager.py

```
@@ -0,0 +1,70 @@
+"""
+Module handling all classification tasks in Logos.
+"""
+import functools
+from typing import List, Tuple
+
+from logos.classification.classify_policy import PolicyClassifier
+from logos.classification.classify_token import TokenClassifier
+from logos.classification.classify_ai import AIClassifier
+from logos.classification.laura_embedding_classifier import LauraEmbeddingClassifier
+from logos.dbutils.dbmanager import DBManager
```

Ruff (0.11.9)

`logos.dbutils.dbmanager.DBManager` imported but unused

Remove unused import: `logos.dbutils.dbmanager.DBManager`

(F401)